aspnetcore/fundamentals/logging/index.md

```diff
@@ -1051,6 +1051,16 @@ logging.AddEventLog();
 * `SourceName` &ndash; ".NET Runtime"
 * `MachineName` &ndash; local machine
 
+Events are logged for [Warning level and higher](https://github.com/aspnet/Extensions/blob/master/src/Hosting/Hosting/src/Host.cs). To log events lower than `Warning`, explicitly set the log level. For example, add the following to the *appsettings.json* file:
```

Comment:
There's a section here that describes them ...

```suggestion
Events are logged for [Warning level and higher](#log-level). To log events lower than `Warning`, explicitly set the log level. For example, add the following to the *appsettings.json* file:
```

If you want ref source tho, the link you have here only covers the Generic Host. Recommend versioning and a link for Web Host if sticking with ref source. `master` also links *next version* ... little risky ... but this stuff is so stable that I doubt it would change ... *maybe*. :smile: